contrib/gorilla/mux/option.go

```diff
@@ -37,6 +38,15 @@ func defaults(cfg *routerConfig) {
 		cfg.serviceName = svc
 	}
 	cfg.resourceNamer = defaultResourceNamer
+	cfg.skippingFunc = defaultSkippingFunc
```

Comment:
... and instead just inline here:
```suggestion
	cfg.ignoreRequest = func(_ *http.Request) { return false }
```